elastic/kibana
# [ResponseOps] Exclude RBAC from SIEM for alerts search strategy

Resolves https://github.com/elastic/kibana/issues/126727

For requests to alerts created by security solutions, we need to exclude RBAC filters as they don't work in the same way as other alerts.

## Testing

1) Create an environment with alerts from SIEM (see https://github.com/elastic/kibana/pull/126813 for help with this).
2) Create a role that can read all ES indices but cannot access security solutions in Kibana
3) Create a user and attach the above role
4) Using curl, try and query for the SIEM alerts
```
POST https://localhost:5601/internal/bsearch
{
	"batch": [
		{
			"request": {
				"featureIds": [
					"siem"
				]
			},
			"options": {
				"strategy": "ruleRegistryAlertsSearchStrategy"
			}
		}
	]
}
```

Ensure `result.rawResponse.hits.total` is more than 0
